instrumentation/otel-webserver-module/test/ApacheTesting.sh

```diff
@@ -11,6 +11,12 @@ if [ $targetSystem = "ubuntu" ]; then
 	OTelApacheModule="/opt/opentelemetry-webserver-sdk/WebServerModule/Apache/libmod_apache_otel.so"
 fi
 
+if [ $targetSystem = "centos7" ]; then
```

Comment:
its better if we move this in line no 10 with LOGICAL OR condition, since the value is same in both case.